content/references-rest-api.md

```diff
@@ -61,6 +61,107 @@ Event data
 | channel | true | A single channel name. *Limited to 100 channels.* |
 | socket_id | false | Exclude a connection from the targets of the event. |
 
+## Channels
+
+### GET channel
+
+Channel writes a JSON object to the response with information about the
+requested channel's occupation. 
+
+```
+/apps/<id>/channel/<name>
+```
+
+#####Request
+
+| Parameter | Required | Description |
+| - | - | - |
+| info | false | User count and subscription count attributes can be requested |
+
+Info attributes
+
+| Attribute | Type | Restrictions | Description |
+| - | - | - |
+| user_count | int | only valid to presence channels | The requested channel's user count |
+| subscription_count | int | - | The requested channel's subscription count |
+
+```
+Example request:
+
+GET /apps/1/channel/presence-test/info=user_count,subscription_count
+```
+
+#####Response
+
+A successful response contains a JSON object. 
+The response optionally contains user count and subscription count attributes if requested.
+
+The response is error (400: Bad request) if the query contains invalid attributes.
+It is invalid to request the user_count attribute for non-presence channels.
+
+```
+Example response:
+
+{
+    "occupied":true,
+    "user_count":1,
+    "subscription_count":1
+}
+```
+
+### GET channels
+
+Channels writes a JSON object to the response with information about the 
+channels.
+
+```
+/apps/<id>/channels
+```
+
+#####Request
+
+| Parameter | Required | Description |
+| - | - | - |
+| filter_by_prefix | false | Channels can be filtered by prefix |
+| info | false | User count attribute can be requested |
+
+Info attributes
+
+| Attribute | Type | Restrictions | Description |
+| - | - | - |
+| user_count | int | only valid to presence channels | The requested channels' user count |
+
+```
+Example request:
+
+GET /apps/1/channels/filter_by_prefix=presence-&info=user_count
+```
+
+#####Response
+
+A successful response contains a JSON object. 
+The response optionally contains presence channels' user count if the user_count attribute is requested
+and only presence channels are filtered.
+
+The response is error (400: Bad request) if the query contains invalid attributes.
```

Comment:
I am not an expert but I think "The response is error" is grammatically incorrect.